Update debug-memory-leak to add now needed step (#47291)
dotnet/docs: docs/core/diagnostics/debug-memory-leak.md

4807 DiagnosticScena /home/user/git/samples/core/diagnostics/DiagnosticScenarios/bin/Debug/netcoreapp3.0/DiagnosticScenarios
```

> [!NOTE]

> > If the above command does not work, or is not found, you likely need to install the `dotnet-counters` tool first, with the following command. 
> 
> ```console
> dotnet tool install --global dotnet-counters
> ```

Reviewer: ```suggestion
> If the previous command doesn't work or isn't found, you likely need to install the `dotnet-counters` tool first. Use the following command:
>
> ```console
> dotnet tool install --global dotnet-counters
> ```
```
